app/assets/javascripts/discourse/app/components/modal/bulk-topic-actions.gjs

```diff
@@ -41,8 +43,8 @@ export default class BulkTopicActions extends Component {
       return this._processChunks(operation);
     } catch {
       this.dialog.alert(i18n("generic_error"));
```

Comment:
Should this line be removed now that we do `showToast` at the end?

Reply:
Probably!